docker/transformers-doc-builder/Dockerfile

```diff
@@ -11,7 +11,6 @@ RUN apt-get -y update && apt-get install -y libsndfile1-dev && apt install -y te
 RUN python3 -m pip install --no-cache-dir ./transformers[deepspeed]
 
 RUN python3 -m pip install --no-cache-dir torchvision git+https://github.com/facebookresearch/detectron2.git pytesseract
-RUN python3 -m pip install --no-cache-dir pytorch-quantization --extra-index-url https://pypi.ngc.nvidia.com
```

Comment:
I don't know if this will affect the doc build for the model `QDQBert`. I tested it locally, the doc build completed without failure, including `transformers/docs/source/en/model_doc/qdqbert.md`.